src/Engine/Airspace/AirspaceWarningManager.cpp

```diff
@@ -28,7 +28,7 @@
 #include "AirspaceAircraftPerformance.hpp"
 #include "Task/Stats/TaskStats.hpp"
 
-#define CRUISE_FILTER_FACT 0.5
+static constexpr float CRUISE_FILTER_FACT = 0.5;
```

Comment:
Same here - `float` is wrong, `double` should be used. In this case, changing it to `double` has probably no effect, because 0.5 can be represented exactly with both types, but for other constants, it may be relevant if using `float` leads to truncation.